sdks/java/core/src/main/java/org/apache/beam/sdk/testing/UsesAdditionalStateParDo.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.testing;
+
+import org.apache.beam.sdk.transforms.ParDo;
+
+/**
+ * Category tag for validation tests which utilize stateful {@link ParDo}
+ * with {@link org.apache.beam.sdk.util.state.SetState} or
+ * {@link org.apache.beam.sdk.util.state.MapState}.
+ */
+public interface UsesAdditionalStateParDo {}
```

Comment:
Can you split this into `UsesSetState` and `UsesMapState`? I know they will usually be implemented at the same time, but it will be very readable in a pom.xml that way.